dist/index.html

```diff
@@ -33,7 +33,7 @@
       if (url && url.length > 1) {
         url = decodeURIComponent(url[1]);
       } else {
-        url = "http://petstore.swagger.io/v2/swagger.json";
+        url = "http://udb-swagger.dev/swagger.json";
```

Comment:
Deze repo is een fork van swagger-ui in de 2dotstwice organization en staat op zich los van udb3. Ik zou hier daarom geen udb3 url als default in zetten.